packages/base/core/src/fonts/oui-icons/oui-icons.html

```diff
@@ -2392,6 +2401,15 @@ <h1>oui-icons</h1>
         </div>
 
 
+        <div class="preview">
+            <span class="inner">
+                <span class=" oui-icon-truck"></span>
+            </span>
+            <br>
+            <span class='label'>truck</span>
+        </div>
+
```

Comment:
Extra WS, maybe you could add a linter to format this automatically

Reply:
This file is generated by the icon-generator, like everything in the `fonts/oui-icons/` folder.
We keep it to see the result of the generated icons.

Reply:
Well we should teach this generator how to write `html` then 😄

Reply:
Do your magic ! https://github.com/Workshape/icon-font-generator/blob/master/template/html.hbs :)